ObjectTK/Buffers/VertexArray.cs

```diff
@@ -88,8 +88,9 @@ public void DrawElements(PrimitiveType mode, int count, DrawElementsType type =
 
         public void DrawElementsIndirect(PrimitiveType mode, DrawElementsType type = DrawElementsType.UnsignedInt, int offset = 0)
         {
-            AssertActive();
-            GL.DrawElementsIndirect(mode, (All)type, new IntPtr(offset));
+            throw new NotImplementedException();
+            //AssertActive();
+            //GL.DrawElementsIndirect(mode, (All)type, new IntPtr(offset));
```

Comment:
This call also takes different parameters now, `ArbDrawIndirect` for both the `mode` and `type` parameters

Reply:
Should be fixed now, the cast was making visual studio show me parameters for a different overload so I didn't realize this one actually did exist.